www/scripts/generate-template-reference.js

```diff
@@ -92,14 +91,11 @@ function getType(prop) {
   return '--';
 }
 
-stdin.on('data', d => {
-  input.push(d);
-});
-
-stdin.on('end', () => {
-  schema = JSON.parse(input.join());
-  writeHeader();
-  writeProperties();
-  writeDefinitions();
-  stdout.write(lines.join('\n'));
+$RefParser.bundle(process.argv[2]).then(function(deref_schema) {
```

Comment:
This isn't anything that you need to change, but just for my own edification, I'm curious about the switch from processing stdin to taking a positional file argument.  Is that just personal preference or is there a different reason for it?

Reply:
If we pipe the file in with stdin we lose the context of the file. References to external files are relative so the library could have a hard time finding them dependent on the directory that the script is run in. This way it doesn't actually mater where you run the script from; it should be able to find its external dependencies regardless... At least that was my theory, I didn't actually test that. Also it just seems cleaner for the script to take an argument instead of reading on stdin :stuck_out_tongue_winking_eye:

Reply:
No, that totally makes sense.  Thanks for the explanation!